docs/index.asciidoc

```diff
@@ -21,6 +21,10 @@ include::{include_path}/plugin_header.asciidoc[]
 
 ==== Description
 
-The JDBC Integration Plugin provides integrated plugins for working with Databases.
+The JDBC Integration Plugin provides the following plugins for working with databases that provide a JDBC interface:
```

Comment:
I'd like to standardize on the wording used for integrations.  Something like:

"The JDBC Integration Plugin provides integrated plugins for working with https://www.oracle.com/technetwork/java/javase/jdbc/index.html[JDBC]."

Reply:
If you decide to include a link, please verify that the sample link I provided is the best resource before using it.

Reply:
@karenzone Is it OK to not include a link? I'm not sure it adds a huge amount of value - it mentions a few different Java database technologies, with a dead link to a JDBC overview...  

Does something like this work?

"The JDBC Integration Plugin provides integrated plugins for working with Databases that provide JDBC drivers?"

Reply:
Absolutely the right call.  If we can't identify a definitive link, then let's skip it. What you are suggesting is fine, which one minor tweak. "Databases" should be lowercase.